### PR TITLE
klatexformula:  Add version 4.0.0

### PR DESCRIPTION
**Pull request by linsui**

### CODE REVIEW

File: bucket/klatexformula.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+            "klatexformula"
+        ]
+    ],
+    "shortcuts": [
+        [
+            "bin\\klatexformula.exe",
+            "kLatexFormula"
+        ]
+    ],
+    "post_install": "Remove-Item \"$dir\\`$PLUGINSDIR\", \"$dir\\Uninstall*\" -Recurse",
```
Comment (Ash258) [inline bucket/klatexformula.json]: pre

File: bucket/klatexformula.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
+{
+    "version": "4.0.0",
+    "homepage": "https://klatexformula.sourceforge.io",
+    "description": "An easy-to-use graphical application for generating images from LaTeX equations.",
+    "license": "GPL-2.0-or-later",
+    "depends": "latex",
```
Comment (Ash258) [inline bucket/klatexformula.json]: Where is mentioned you need to have miktex installed?

Comment (linsui) [reply bucket/klatexformula.json]: https://klatexformula.sourceforge.io/downloads#system-requirements

Comment (Ash258) [reply bucket/klatexformula.json]: > On Windows, you **can** install

Why do you force user to have Miktex installed? Why I cannot use installation of Texlive?

Comment (linsui) [reply bucket/klatexformula.json]: I'll change it to suggest.

**Pull request merged**